.github/release.yaml

```diff
@@ -11,30 +11,17 @@ changelog:
     - title: Exciting New Features 🎉
       labels:
         - enhancement
-    - title: Lambda Functions
-      labels:
-        - lambda
     - title: Bug Fixes 🐜
       labels:
         - bug
         - hotfix
     - title: Security Fixes 🔐
       labels:
-        - cve
         - security
-    - title: DevOps
-      labels:
-        - actions
-        - devops
-    - title: Dependency Upgrades 🆙
-      labels:
-        - dependencies
     - title: Internal Upgrades 📈
       labels:
         - internal
-        - tech debt
-        - tool
-        - maintenance
-    - title: Other Changes 🧑‍💻
+    - title: Other Changes
```

Comment:
You removed the emoji for other changes? 😞

Reply:
Good catch - even though we hope to never have some label other than the above, we still want to keep the emoji, plz